stock_picking_type_analytic/models/stock_picking.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2022 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class StockPicking(models.Model):
+
+    _inherit = "stock.picking"
+
+    @api.model
+    def create(self, vals):
+        """
+        Get analytic account from picking type if set
+        """
+        defaults = self.default_get(["name", "picking_type_id"])
+        picking_type = self.env["stock.picking.type"].browse(
+            vals.get("picking_type_id", defaults.get("picking_type_id"))
+        )
+        if "analytic_account_id" not in vals and picking_type.analytic_account_id:
+            vals.update(
+                {
+                    "original_analytic_account_id": picking_type.analytic_account_id.id,
```

Comment:
Shouldn't it be analytic_account_id instead of original_analytic_account_id?

Reply:
https://github.com/OCA/account-analytic/blob/14.0/stock_picking_analytic/models/stock_picking.py#L11

Reply:
But indeed, this highlight the fact tests don't pass on that line.